docs/source/admin/traffic_ops/using.rst

```diff
@@ -296,30 +296,33 @@ The fields in the Delivery Service view are:
+| DNS Bypass CNAME                                 |                                                                                                                                                                                                                     |
 +--------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Query String Handling                            | How to treat query strings:                                                                                                                                                                                         |
-|                                                  |                                                                                                                                                                                                                     |
-|                                                  | - 0 use in cache key and hand up to origin -this means each unique query string Is treated as a unique URL.                                                                                                         |
-|                                                  | - 1 Do not use in cache key, but pass up to origin - this means a 2 URLs that are the same except for the query string will match, and cache HIT, while the origin still sees original query string in the request. |
-|                                                  | - 2 Drop at edge - this means a 2 URLs that are the same except for  the query string will match, and cache HIT, while the origin will not see original query string in the request.                                |
-|                                                  |                                                                                                                                                                                                                     |
-|                                                  | **Note:** Choosing to drop query strings at the edge will preclude the use of a Regex Remap Expression. See :ref:`rl-regex-remap`.                                                                                  |
-|                                                  | To set the qstring without the use of regex remap, or for further options, see :ref:`rl-qstring-handling`.                                                                                                          |
+| DNS Bypass Ip                                    | (For DNS routed delivery services only) This is the address to respond to A requests with when the the max Bps or Max Tps for this delivery service are exceeded.                                                   |
```

Comment:
"the the"

Reply:
oh man, i'm getting dinged for previous errors. no worries. i'll fix